src/components/ReportActionItem/MoneyReportView.tsx

```diff
@@ -57,116 +67,135 @@ function MoneyReportView({report, policy}: MoneyReportViewProps) {
+                                        <Text
+                                            style={[styles.textLabelSupporting]}
+                                            numberOfLines={1}
+                                        >
+                                            {translate('cardTransactions.outOfPocket')}
+                                        </Text>
+                                    </View>
+                                    <View style={[styles.flexRow, styles.justifyContentCenter]}>
+                                        <Text
+                                            numberOfLines={1}
+                                            style={subAmountTextStyles}
+                                        >
+                                            {formattedOutOfPocketAmount}
+                                        </Text>
+                                    </View>
                                 </View>
-                                <View style={[styles.flexRow, styles.justifyContentCenter]}>
-                                    <Text
-                                        numberOfLines={1}
-                                        style={subAmountTextStyles}
-                                    >
-                                        {formattedCompanySpendAmount}
-                                    </Text>
+                                <View style={[styles.flexRow, styles.pointerEventsNone, styles.containerWithSpaceBetween, styles.ph5, styles.pv1]}>
+                                    <View style={[styles.flex1, styles.justifyContentCenter]}>
+                                        <Text
+                                            style={[styles.textLabelSupporting]}
+                                            numberOfLines={1}
+                                        >
+                                            {translate('cardTransactions.companySpend')}
+                                        </Text>
+                                    </View>
+                                    <View style={[styles.flexRow, styles.justifyContentCenter]}>
+                                        <Text
+                                            numberOfLines={1}
+                                            style={subAmountTextStyles}
+                                        >
+                                            {formattedCompanySpendAmount}
+                                        </Text>
+                                    </View>
                                 </View>
-                            </View>
-                        </>
-                    )}
-                </>
-            )}
-        </View>
+                            </>
+                        )}
+                    </>
+                )}
+            </View>
+            <SpacerView
+                shouldShow={!shouldHideThreadDividerLine}
```

Comment:
Seems we missed considering `isCombinedReport` here which caused issue https://github.com/Expensify/App/issues/58015